docs/usage_modes.rst

```diff
@@ -0,0 +1,147 @@
+functions as a set of instructions that create a "container" capable of executing a
+specified application on the Docker platform. This is useful when an application
+requires substantial setup or processing prior to use. The PUDL Docker image outlines
+the steps necessary to run the ETL so that users don't have to do it themselves every
+time they want to access the processed data or use PUDL functions.  Users can choose to
+interact directly with the processed PUDL data, through SQL or the parquet files, or run
+the Docker image to use the data within a Docker container. Using Docker provides the
+user with access to all of the denormalized output tables and analysis functions
+available through the PUDL software and is recommended for users seeking greater
+analysis capabilities.
+
+To begin using Docker, you'll need to
+`install Docker & docker-compose <https://docs.docker.com/get-docker/>`_,
+download this big compressed tarball, extract the tarball in a directory on your
+computer, and run docker image load.
+
+.. code-block:: console
+
+   $ EXAMPLE
+   $ docker image load --input filename.tar
+
+
+Make sure you don't have another Jupyter Notebook server running on port 8888 already by
+running the foreground commend ``fg``. If you do, press ``ctr c`` to quit. Continue
+testing the ``fg`` command until it outputs ``no such job``.
+
+.. code-block:: console
+
+   $ fg
+
+Run docker-compose up (need to refer to the just-loaded image, not the docker hub
+reference) Visit the URL it prints out which should start with:
+``https://127.0.0.1:8888`` or ``https://localhost:8888``
+
+.. code-block:: console
+
+    $ docker-compose up
+
+Opening the docker container will open PUDL’s JupyterLab example notebooks:
+
+* `Accessing PUDL SQLite tables <https://github.com/catalyst-cooperative/pudl-tutorials/blob/main/notebooks/01-pudl-database.ipynb>`_
+* `Accessing PUDL’s de-normalized and analysis tables <https://github.com/catalyst-cooperative/pudl-tutorials/blob/main/notebooks/02-pudl-outputs.ipynb>`_
+* `Accessing the larger-than-memory PUDL data stored in parquet files <https://github.com/catalyst-cooperative/pudl-tutorials/blob/main/notebooks/03-pudl-parquet.ipynb>`_
+
+.. _jup-hub:
+
+JupyterHub
+^^^^^^^^^^
+
+PUDL’s `JupyterHub <https://jupyter.org/hub>`_ is currently experimental. It has access
+to all of PUDL’s fully processed data, the PUDL example notebooks enumerated in the Data
+Release section above and computational resources.
+
+This setup is similar to using the Zenodo archives with the Docker container, but it
+avoids downloading the pre-processed data and navigating Docker. We plan to updating the
+data we are providing access to via JupyterHub more often than we will publish new data
+archives on Zenodo.
+
+To access the JupyterHub, you'll need to:
+
+* Submit a `request form <https://docs.google.com/forms/d/1i8O5kbExFkR-urrmYPdWSIDl-Aub-nx5eMFgOn6TTnM/edit>`_
```

Comment:
I think this is the link to edit the form not the link to submit it. I think I put this in there so my bad.

Here is the user facing link:
https://forms.gle/TN3GuE2e2mnWoFC4A

Reply:
I updated all the other links I could find to use this one. But it seems to be sending the messages to my spam folder. :-/